src/SlotUsageCheque/SlotUsageCheque.model.ts

```diff
@@ -1,7 +1,23 @@
-import { Model } from 'decentraland-server'
+import { Model, raw, SQL } from 'decentraland-server'
+import { Collection } from '../Collection'
 
 import { SlotUsageChequeAttributes } from './SlotUsageCheque.types'
 
 export class SlotUsageCheque extends Model<SlotUsageChequeAttributes> {
   static tableName = 'slot_usage_cheques'
+
+  static async findLastByCollectionId(
+    collectionId: string
+  ): Promise<SlotUsageChequeAttributes | undefined> {
+    const result = await this.query<SlotUsageChequeAttributes>(SQL`
+      SELECT slot_usage_cheque.*
+      FROM ${raw(this.tableName)} slot_usage_cheque
+      INNER JOIN ${raw(
+        Collection.tableName
+      )} collection ON collection.id = slot_usage_cheque.collection_id AND collection.collection_id = ${collectionId} 
```

Comment:
```suggestion
      )} collection ON collection.id = slot_usage_cheque.collection_id AND collection.id = ${collectionId} 
```
I think you want `collection.id` instead of `collection.collection_id`